src/App.js

```diff
@@ -1,5 +1,28 @@
+import { EXECUTION_RESULT, FINAL_WINNER, INPUT } from "./constants/race.js";
+import { simulateRace } from "./race/race.js";
+import { parseCarNames } from "./car/parseCarNames.js";
+import { attemptValidator } from "./validation/attemptValidator.js";
+import { getInput } from "./view/input.js";
+import { getOutput, getResult, getWinner } from "./view/output.js";
+
 class App {
-  async run() {}
+  async run() {
+    try {
+      const carNamesInput = await getInput(INPUT.RACE_CAR_NAME);
+      const carNames = parseCarNames(carNamesInput);
+      const attemptCountInput = await getInput(INPUT.ATTEMPT_COUNT_PROMPT);
+      const attemptCount = attemptValidator(attemptCountInput);
```

Comment:
parseCarNames 함수와 attemptValidator 함수의 역할은 유효성을 검증하는 함수로 보여요!

하지만 다른 이름을 가지고 있어서 마치 다른 동작을 하는 함수처럼 느껴지는 거 같아요

일관된 함수 이름을 사용하면 어떨까요?
```suggestion
      const carNamesInput = await getInput(INPUT.RACE_CAR_NAME);
      const carNames = validateCarNames(carNamesInput);
      
      const attemptCountInput = await getInput(INPUT.ATTEMPT_COUNT_PROMPT);
      const attemptCount = validateAttemptCount(attemptCountInput);
```